app/views/admin/cms/_collection_list.html.erb

```diff
@@ -0,0 +1,41 @@
+<table>
+  <thead>
+    <tr>
+      <th>Collection ID</th>
+      <th>Title</th>
+      <th>Items</th>
+      <th>Published</th>
+      <th>Actions</th>
+      <th></th>
+    </tr>
+  </thead>
+
+  <tbody id="collection_list">
+    <% Collection.all.each do |resource| %>
```

Comment:
By using resource as a generic reference to the model instance I am able to use the shared errors partial with the same reference for all model errors. I'm sure that there's another way of doing it though :)